samples/immersed_boundary/writeVTK.py

```diff
@@ -0,0 +1,19 @@
+def WriteVTK(system, outFile):
```

Comment:
There is `system.part.writevtk`.

https://github.com/espressomd/espresso/blob/37dd602d01b73109804f2fdb14c70763768767ee/src/python/espressomd/particle_data.pyx#L1838-L1867